packages/frontend/amp/server/render.tsx

```diff
@@ -26,6 +26,9 @@ export const render = ({ body }: express.Request, res: express.Response) => {
                       config.subscribeWithGoogleApiUrl,
                   )
                 : []),
+            ...[
```

Comment:
Should this be in `scripts.ts`?

Reply:
Ok... Interesting one. 

Here is the situation:

`scripts.ts` has logic to include AMP scripts per element types. This is not exactly what we have here. On the other hand `document.tsx` has default (always present scripts), and this is not what we have either. What we have a here is a script that should be added on specific articles (eg live blogs). 

As a **temporary**( measure (and talking about this with @MatthewJWalls ) I have put it in a new section on `document.tsx`. In a separate piece of work we will address the issue of unambiguously detect liveblogs and add the required (cleaner) logic for adding live blog specific scripts such as this one :)

Reply:
Yup, ok, can you add a todo link to this comment? :)

Reply:
Done! :)